src/E0_evaluation.py

```diff
@@ -8,14 +8,32 @@
 import src.E3_indicator_calculation as indicators
 import src.F0_output as output
 
+r"""
+Module E0 evaluation
+====================
+
+Module E0 evaluates the oemof results and calculates the KPI
+- define dictionary entry for kpi matrix
+- define dictionary entry for cost matrix
+- store all results to matrix
```

Comment:
Are theses things something you can test here? If kpi/kpi_matrix and kpi/cost_matrix are keys in dict values, and if dict_values/kpi/kpi_matrix and dict_values/kpi/cost_matrix are pd.DataFrames?

Reply:
Are you sure "kpi_matrix" is a key? At least the test I write to check that failed ...

Reply:
Oh, sorry, it is "scalar_matrix".